tools/debugger/debugger_core.py

```diff
@@ -844,6 +906,19 @@ def _send_breakpoint(self, breakpoint):
                               breakpoint.offset)
         self.channel.send_message(self.byte_order, message)
 
+    def _send_pending_config(self, enable):
+        if enable:
+            message = struct.pack(self.byte_order + "BBB",
+                                  1 + 1,
+                                  ESCARGOT_DEBUGGER_PENDING_CONFIG_ENABLE,
+                                  enable)
+        else:
+            message = struct.pack(self.byte_order + "BBB",
+                                  1 + 1,
+                                  ESCARGOT_DEBUGGER_PENDING_CONFIG_DISABLE,
+                                  enable)
```

Comment:
I don't get this. We have `ESCARGOT_DEBUGGER_PENDING_CONFIG_ENABLE` / `ESCARGOT_DEBUGGER_PENDING_CONFIG_DISABLE` but we also send `enable` as well? Perhaps we could have only a single `ESCARGOT_DEBUGGER_PENDING_CONFIG` message then.